App/features/group/reducer.ts

```diff
@@ -4,19 +4,22 @@ import { ActionType } from 'typesafe-actions'
 import { feedReducer, FeedState } from './feed'
 import { addMessageReducer, AddMessageState } from './add-message'
 import { addPhotoReducer, ProcessingImagesState } from './add-photo'
+import { renameGroupReducer, RenameGroupState } from './rename-group'
 
 import * as actions from './actions'
 
 export interface GroupState {
   feed: FeedState
   addMessage: AddMessageState
   addPhoto: ProcessingImagesState
+  renameGroup: RenameGroupState
 }
 
 export type GroupAction = ActionType<typeof actions>
 
 export default combineReducers<GroupState>({
   feed: feedReducer,
   addMessage: addMessageReducer,
-  addPhoto: addPhotoReducer
+  addPhoto: addPhotoReducer,
+  renameGroup: renameGroupReducer
```

Comment:
Good work groking the nested feature setup ;)

Reply:
Thanks 😄.